### ISSUE TITLE
Single source of truth for documentation

### ISSUE THREAD

**Issue by jennyknuth** [SWxTREC/vector-code#17]
this file: https://github.com/SWxTREC/vector-code/blob/master/docs/DOCUMENTATION.md is now redundant with this file: https://github.com/SWxTREC/vector-api/blob/master/README.md. 

Since the content of this documentation now goes with the API, I suggest we either remove this file or change its purpose and content to document the model code. 

If at some point the the model code is documented, that information can go here and we can once again display it on the front end.

**Comment by greglucas** [comment]
I agree, this is not needed anymore because this "code" repo doesn't have any API information and it isn't needed. It should probably be removed altogether.

Your suggestion about actually documenting the model code is a good one, which should probably be in the README file, not a separate DOCUMENTATION file.
